Herebyfile.mjs

```diff
@@ -313,7 +314,34 @@ function entrypointBuildTask(options) {
         run: () => buildProject(options.project),
     });
 
-    const bundler = createBundler(options.srcEntrypoint, options.output, options.bundlerOptions);
+    const mainDeps = options.mainDeps?.slice(0) ?? [];
+
+    let output = options.output;
+    if (options.enableCompileCache) {
+        const originalOutput = output;
+        output = path.join(path.dirname(output), "_" + path.basename(output));

```

Comment:
This was just an expedient name choice; I can also remove the extension and then make it like `tsc_.js` which may sort better.

Reply:
I was going to say that `_tsc` tab-completes better on the command-line, though, because it doesn't interfere with `types<TAB>` -- but there's so much stuff in built/local that it doesn't actually make a difference.